hvcc/generators/c2js/c2js.py

```diff
@@ -185,6 +190,39 @@ def compile(clazz, c_src_dir, out_dir, externs,
                     events=event_list,
                     copyright=copyright_html))
 
+            # generate heavy js worklet from template
+            # Note: this file will be incorporated into the emscripten output
+            # and removed afterwards
+            post_js_path = os.path.join(out_dir, "hv_worklet.js")
+            with open(post_js_path, "w") as f:
+                f.write(env.get_template("hv_worklet.js").render(
+                    name=patch_name,
+                    copyright=copyright_js,
+                    externs=externs,
+                    pool_sizes_kb=externs["memoryPoolSizesKb"]))
+
+            pre_js_path = os.path.join(out_dir, "hv_worklet_start.js")
+            with open(pre_js_path, "w") as f:
+                f.write(env.get_template("hv_worklet_start.js").render(
+                    name=patch_name,
+                    copyright=copyright_js,
+                    externs=externs,
+                    pool_sizes_kb=externs["memoryPoolSizesKb"]))
+
+            js_path = c2js.run_emscripten(c_src_dir=c_src_dir,
+                                          out_dir=out_dir,
+                                          patch_name=patch_name,
+                                          output_name=patch_name+"_AudioLibWorklet",
```

Comment:
please use f-strings for combining variables and strings. so this would be `f"{patch_name}_AudioLibWorklet"`

Reply:
Should be good on this now